onnxruntime/core/providers/js/js_data_types.cc

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/providers/cpu/tensor/shape_op.h"
+
+namespace onnxruntime {
+namespace js {
+
+using SupportedTypes =
+    TypeList<
+        float,
+        int32_t,
+        uint32_t>;
+
+std::vector<MLDataType> JsepSupportedDataTypes() {
```

Comment:
nit: is it possible to make this const so no need to generate the std vector every time?

Reply:
Sure